app/controllers/proposals_controller.rb

```diff
@@ -1,33 +1,33 @@
 class ProposalsController < ApplicationController
   include ApplicationHelper
   before_action :authenticate_user!, :except => [:show, :index, :related_vote_in_tree, :related_proposals]
-  before_action :find_hub, :find_user, only: :index
 
   # GET /proposals.json
   def index
```

Comment:
Sounds fantastic, Steve. If you felt like adding those paths, I would be more than happy to change the client side. Consequences of changing API are limited at this time, so there really isn't a better time to do it.

Actually, the CoffeeScript code is incredibly simple to change there:
```coffeescript
 $scope.setFilter = (filterSelected) ->
    $location
      .search 'filter', filterSelected

  $scope.setHub = (hubSelected) ->
    $location
      .path '/proposals/' 
      .search 'hub', hubSelected.id
```
So for example, this   
```coffeescript
  $scope.setHub = (hubSelected) ->
    $location
      .path '/proposals/' 
      .search 'hub', hubSelected.id
```
... becomes this this:
```coffeescript
  $scope.setHub = (hubSelected) ->
    $location
      .path '/' + hubSelected.id + '/proposals/' 
```
There are a few more places to change it, but all cases are the same syntax and easily found by searching the .coffee's for '$location'.